integration/config/core.yaml

```diff
@@ -458,7 +466,7 @@ vm:
 
         # Enables/disables the standard out/err from chaincode containers for
         # debugging purposes
-        attachStdout: true
+        attachStdout: false
```

Comment:
Let's keep this `true` as we may need it since shim.logger has been removed and we need to handle chaincode logging manually.